### PR TITLE
[JENKINS-68195] Duplicated help icon for Host Key verification

### PR DESCRIPTION
**Pull request by kuisathaverat**
See [JENKINS-68195](https://issues.jenkins-ci.org/browse/JENKINS-68195).

### Submitter checklist

- [X] JIRA issue is well described
- [X] Appropriate autotests or explanation to why this change has no tests

After some improvements in the `f:dropdownDescriptorSelector` and the UI present in Jenkins Core 2.242 I need to change a couple of things in some jelly files. I have removed a workaround to show the help link on `f:dropdownDescriptorSelector` and I have to put the `f:dropdownDescriptorSelector` inside an `<f:entry>` to avoid showing the help button in the end of the right side of the browser. This is the visual result of the changes:
Before:
Screenshot 2022-04-08 at 11 41 18
After:
Screenshot 2022-04-08 at 11 17 03

Related to https://github.com/jenkinsci/jenkins/pull/5417

## Note
This PR cannot be merged before Jenkins Core 2.42 (or upper version) is released as LTS because it breaks the help links in previous versions.

### CODE REVIEW

File: src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly [hunk 1]
```diff
@@ -5,7 +5,9 @@
       <c:select checkMethod="post"/>
   </f:entry>
 
-  <f:dropdownDescriptorSelector title="${%Host Key Verification Strategy}" field="sshHostKeyVerificationStrategy" />
+  <f:entry>
```
Comment (mawinter69) [inline src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly:8]: I think this is not really necessary when https://github.com/jenkinsci/jenkins/pull/6255 gets merged. There I faced the same issue and the solution was that the form needs the `class="jenkins-form"`

File: pom.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
   <properties>
     <revision>1</revision>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.319.1</jenkins.version>
```
Comment (kuisathaverat) [inline pom.xml:30]: TODO: set proper LTS version before merge

**Review by jglick — COMMENTED** [review]
Seems to relate to GUI changes I have not been following, so I am likely unqualified to review.

File: pom.xml (hunk 2 above)
Comment (timja) [reply pom.xml:30]: Just bump to weekly? can always backport important changes?

File: src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly (hunk 1 above)
Comment (timja) [reply src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly:8]: worth waiting for this 👍

**Review by timja — APPROVED** [review]
nice

File: src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly (hunk 1 above)
Comment (jglick) [reply src/main/resources/hudson/plugins/sshslaves/SSHConnector/config.jelly:8]: So, revert this hunk and update to 2.343?

File: pom.xml [hunk 3]
```diff
@@ -27,7 +27,7 @@
...
+    <jenkins.version>2.346.2</jenkins.version>
```
Comment (jglick) [inline pom.xml:30]: Note that 2.346.1 is recommended: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions

**Pull request merged**